packages/private-build-infra/src/utilities/unique-add.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function uniqueAdd(obj, key, values) {
```

Comment:
This should die 😄 (we can use `Set`'s)

Reply:
that might be messier given the format that the babel plugin expects is an array